extensions/random-data-generator/src/index.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  Icon,
+  List,
+  OpenInBrowserAction,
+  PasteAction,
+  getPreferenceValues,
+  allLocalStorageItems,
+  setLocalStorageItem,
+} from '@raycast/api';
+import faker from 'faker';
+import _ from 'lodash';
+import isUrl from 'is-url';
+import { useCallback, useEffect, useState } from 'react';
+
+type LocalStorageValues = {
+  pinnedItemIds: string;
+};
+
+type Preferences = {
+  locale: string;
+};
+
+type Item = {
+  section: string;
+  id: string;
+  value: string;
+  getValue(): string;
+};
+
+type Pin = (item: Item) => void;
+
+const blacklistPaths = [
+  'locales',
+  'locale',
+  'localeFallback',
+  'definitions',
+  'fake',
+  'unique',
+  'helpers',
+  'mersenne',
+  'random',
+];
+
+const { locale }: Preferences = getPreferenceValues();
+faker.locale = locale;
+
+const buildItems = (path: string) => {
+  return _.reduce(
+    path ? _.get(faker, path) : faker,
+    (acc: Item[], func, key) => {
+      if (blacklistPaths.includes(key)) {
+        return acc;
+      }
+
+      if (_.isFunction(func)) {
+        const getValue = (): string => func().toString();
+        acc.push({ section: path, id: key, value: getValue(), getValue });
+      } else {
+        acc.push(...buildItems(path ? `${path}.${key}` : key));
+      }
+
+      return acc;
+    },
+    []
+  );
+};
+
+const items = buildItems('');
+const groupedItems = _.groupBy(items, 'section');
+
+function FakerListItem(props: { item: Item; pin?: Pin; unpin?: Pin }) {
+  const { item, pin, unpin } = props;
+  const [value, setValue] = useState(item.value);
+
+  return (
+    <List.Item
+      key={item.id}
```

Comment:
This `key` is no needed here 😄

Reply:
```suggestion
```

key is not a property in List.Item - as @vimtor stated, just remove it 😄